utilities/indexer.cfm

```diff
@@ -107,7 +107,7 @@
 	<cfset vData = {"name"="Lucee #v# New Functions and Tags","type"="listing","description"="List of tags and functions added in Lucee #v#", "related"=lucee_versions[v]}>
 	<cfset arraySort(vData.related, "text")>
 	<cfset fileWrite(dataDir & "/lucee#reReplace(v, "[^0-9]", "", "ALL")#.json", prettyJSON(serializeJSON(vData), "utf-8"))>
-	<p>Wrote lucee<cfoutput>#v#</cfoutput>.json</p>
+	<p>Wrote Lucee<cfoutput>#v#</cfoutput>.json</p>
```

Comment:
Shouldn't this one stay lowercase since it's the file name?

Reply:
Thanks 👍